app/models/translation_string.rb

```diff
@@ -59,11 +59,11 @@ def update_version?
   end
 
   def self.translations(locale)
-    I18n.backend.load_translations if I18n.backend.send(:translations).empty?
-    I18n.backend.send(:translations)[locale.to_sym][MO.locale_namespace.to_sym]
+    # NOTE: This method now checks if it's been initialized, i.e. not empty.
+    I18n.backend.translations[locale.to_sym][MO.locale_namespace.to_sym]
```

Comment:
Note: I restored this check in a subsequent PR. I had misread the I18n method — it has a `translations(do_init: boolean)` parameter